### DISCUSSION TITLE
can we expect support for Rust in future?

### DISCUSSION THREAD

**Discussion by C0DE-IN** [grpc/grpc-web#1411] [General]
can we expect support for Rust in future?

**Comment by sampajano** [comment]
@C0DE-IN Hi! gRPC-Web (this project) is targeted at the web platform (e.g. browsers), so Rust is not relevant in our scope.

Do you mean for gRPC to support Rust in general?

In that case, you could start an discussion at https://groups.google.com/g/grpc-io or https://github.com/grpc/grpc.

Thanks!
